Jamie_Keng/skillset_challenge.ipynb

```diff
@@ -0,0 +1,595 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# <center> Statistical Arbitrage with Vine Copulas - Pair Selection </center>\n",
+    "\n",
+    "Author: Jui Lin(Jamie) Keng <br>\n",
+    "Email: jamiekeng1016@gmail.com <br>\n",
+    "LinkedIn : https://www.linkedin.com/in/jui-lin-jamie-keng-3883b6131/"
+   ]
+  },
+  {
+   "cell_type": "markdown",
```

Comment:
Why does your implementation require the user to perform basic processing? This should be handled internally. 

Ideally, a user would import a class and then can call as few methods as possible to get to their end goal.